### PR TITLE
add olap guide

### PR DESCRIPTION
**Pull request by ChuckHend**
Add guide based on clickbench data

### CODE REVIEW

**Review by DarrenBaldwin07 — APPROVED** [review]

File: docusaurus/docs/tembo-stacks/olap.md [hunk 1]
```diff
@@ -4,27 +4,248 @@ sidebar_position: 6
 
 # Tembo OLAP
 
-Postgres tuned for online analytical processing. Deploy a stack that is optimized for large data sets, complex queries, and high throughput.
```
Comment (xenophenes) [inline docusaurus/docs/tembo-stacks/olap.md:7]: Why are the "Configuration" and "Extension" sections being removed? Is it the plan to eventually remove these sections across the documentation for all Stacks?

If so, why?? & who all is involved in the decision? @ChuckHend 

Developer specs are important for transparency and a simple breakdown of the software-added value of the solution

File: docusaurus/docs/tembo-stacks/olap.md [hunk 2]
```diff
@@ -4,27 +4,248 @@ sidebar_position: 6
...
-- [postgres_fdw] (https://pgt.dev/extensions/postgres_fdw) - `postgres_fdw` provides the foreign data wrapper necessary to access data stored in external Postgres servers. 
-- [clerk_fdw] (https://pgt.dev/extensions/clerk_fdw) -  `clerk_fdw` provides the foreign data wrapper necessary to access data stored in external Clerk servers. 
-- [redis_fdw] (https://pgt.dev/extensions/redis_fdw) - `redis_fdw` provides the foreign data wrapper necessary to access data stored in external Redis servers. 
-- Extensions from [Trunk](https://pgt.dev) can be installed on-demand.
+Run analytical workloads on a Postgres instance tuned for online analytical processing. Build applications that require complex queries and aggregations on large datasets.
+
+## Getting Started
+
+This guide will walk through setting up an analytical workload on Postgres using [Clickbench's "hits" dataset](https://github.com/ClickHouse/ClickBench?tab=readme-ov-file#history-and-motivation) and the [columnar](https://github.com/hydradatabase/hydra) Postgres extension. The dataset represents page views on a system and the guide will demonstrate using extensions to optimize the database for analytical queries.
```
Comment (xenophenes) [inline docusaurus/docs/tembo-stacks/olap.md:11]: > The dataset represents page views on a system and the guide will demonstrate using extensions to optimize the database for analytical queries.

should be

> The dataset represents page views on a system. This guide will demonstrate using extensions how users can optimize the database for analytical queries.

(hope you don't mind I'm not directly committing minor changes this go around... will do so next time)

File: docusaurus/docs/tembo-stacks/olap.md (hunk 1 above)
Comment (xenophenes) [reply docusaurus/docs/tembo-stacks/olap.md:7]: If the concern is the length of the doc page, could always split it up like


< Stack name >
- Technical Specs
- Getting Started


for each Stack under "Tembo Stacks"

File: docusaurus/docs/tembo-stacks/olap.md [hunk 3]
```diff
@@ -4,27 +4,248 @@ sidebar_position: 6
...
+
+### Create a Tembo OLAP Stack instance
+
+Navigate to [cloud.tembo.io](https://cloud.tembo.io) and create a new stack. Select the "OLAP" Stack type and choose the desired instance size. This guide will function on the hobby tier, but it is recommended to use a larger instance for production workloads.
+
+Connect to the instance using `psql`.
+
+The `columnar` extension should be pre-installed and enabled on the `postgres` database.
```
Comment (xenophenes) [inline docusaurus/docs/tembo-stacks/olap.md:19]: Any Tembo recommended methods of installation? At least link to installation instructions that may be relevant here for ease of user experience

File: docusaurus/docs/tembo-stacks/olap.md [hunk 4]
```diff
@@ -4,27 +4,248 @@ sidebar_position: 6
...
+\copy hits FROM 'hits_1mil.tsv'
+```
+
+```text
+COPY 1000000
+```
+
+### Create indices on the table
+
+Create indices on the table to optimize the workload for analytical queries.
```
Comment (xenophenes) [inline docusaurus/docs/tembo-stacks/olap.md:173]: Why does creating indices optimize the workload for analytical queries? Provide a brief ELI5 description for the reader - 1-2 sentences explaining would be fine

File: docusaurus/docs/tembo-stacks/olap.md [hunk 5]
```diff
@@ -4,27 +4,248 @@ sidebar_position: 6
...
+```sql
+CREATE INDEX userid on hits (UserID);
+CREATE INDEX eventtime on hits (EventTime);
+CREATE INDEX eventdate on hits (EventDate);
+CREATE INDEX search on hits (SearchPhrase);
+CREATE INDEX search2 on hits (SearchPhrase) WHERE SearchPhrase <> ''::text;
+```
+
+### Execute analytical queries on the table
+
```
Comment (xenophenes) [inline docusaurus/docs/tembo-stacks/olap.md:184]: Provide some transitional sentence here, something like

> At this point, you are able to execute typical analytical queries and should observe a noticeable improvement in performance given the optimizations you just enabled. Here's a few examples of queries you can try.

Then, make the example titles (i.e. "Get the minimum and maximum event dates from the table.") either bold or a bottom-level header.

**Review by xenophenes — CHANGES_REQUESTED** [review]
Hey @ChuckHend, all finished here, I mostly am interested in an answer for the first question as I strongly think we should either retain & update these sections or separate them into a "Technical Specs" sub-page, same-level page as this "Getting Started" guide. Getting started information is important, but so is technical specs for devs. 

Eventually material could be pulled together in a "Contributing" guide but I think specs should still be associated as that's most likely where devs will check first for more info on additional features & value points (What does this PG wrapper add in value?). 

Thanks - let me know if a call/chat would be helpful, I'll be around after 11 AM PST

File: docusaurus/docs/tembo-stacks/olap.md (hunk 3 above)
Comment (ChuckHend) [reply docusaurus/docs/tembo-stacks/olap.md:19]: It comes pre-installed on this Stack. The user does need need to do anything.

File: docusaurus/docs/tembo-stacks/olap.md (hunk 1 above)
Comment (ChuckHend) [reply docusaurus/docs/tembo-stacks/olap.md:7]: Let's split it into multiple pages per stack. I think it makes sense to have the technical specs and configuration details separated. It would be ideal if we can link out to the source code for those configurations since they do change fairly regularly at this stage in their lifecycle.

Another reason for splitting is that I think we should make a place for multiple how-to style guides.

**Comment by ChuckHend** [comment]
@xenophenes - I added a second page, "technical specs" dedicated to configuration details and other reference. Keep in mind the source of truth lives in the [tembo-stacks](https://github.com/tembo-io/tembo/tree/main/tembo-stacks) project repo, so having this type of organization will require us to be diligent in keeping this repo up-to-date as the Stacks evolve. In the future we may want to render the documentation from the source instead.

Overall though I think splitting these Stack docs into multiple pages is going to be necessary.
